backend/pkg/commons/db2/raw/client_test.go

```diff
@@ -0,0 +1,307 @@
+package raw
+
+import (
+	"context"
+	"math/big"
+	"net/http"
+	"os"
+	"testing"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethereum/go-ethereum/rpc"
+
+	"github.com/gobitfly/beaconchain/pkg/commons/db2/database"
+	"github.com/gobitfly/beaconchain/pkg/commons/db2/databasetest"
+)
+
+const (
+	chainID uint64 = 1
+)
+
+func TestBigTableClientRealCondition(t *testing.T) {
+	project := os.Getenv("BIGTABLE_PROJECT")
+	instance := os.Getenv("BIGTABLE_INSTANCE")
+	if project == "" || instance == "" {
+		t.Skip("skipping test, set BIGTABLE_PROJECT and BIGTABLE_INSTANCE")
+	}
+
+	tests := []struct {
+		name  string
+		block int64
+	}{
+		{
+			name:  "test block",
```

Comment:
The test names could be more descriptive - eg this one we can consider renaming to "Test Block Retrieval" or something similar for better clarity

Reply:
I prefer to keep it simple.
`TestBigTableClient.test_block` provide the same amount of info than `TestBigTableClient.Test_Block_Retrieval`
Or we could rename the test itself rather than the case.
But `TestBigTableClient` is describing what the test does, so I don't really how to improve it